pcsx2/R5900OpcodeImpl.cpp

```diff
@@ -883,8 +883,10 @@ void SYSCALL()
 
 	BIOS_LOG("Bios call: %s (%x)", R5900::bios[call], call);
```

Comment:
Does it compile in debug mode ? What happen if you enable the bios log trace (in the F10 menu)